book_source/workflow/inputs_conversions.Rmd

```diff
@@ -0,0 +1,27 @@
+# Input Data
```

Comment:
This whole page seem somewhat redundant with, but less developed than, https://pecanproject.github.io/pecan-documentation/develop/basic-model-configuration.html

Reply:
Yeah definitely redundant as of now.  That's why it will not be listed in the `bookdown.yml` so it will not get rendered. At some point though this will be the header page for the "Inputs" section of the "Workflow" section of documentation.